### PR TITLE
Adding real legends in printing

### LINKED ISSUE #591: integrate legend printing in application
Legends are now printed, but they are hard-coded

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #591

Please review.

### CODE REVIEW

**Comment by jaykayone** [comment]
The system now prints the correct legends!

.. but:

it also does so when the "print legends" checkbox is not checked

**Comment by pgiraud** [comment]
Indeed. Pull request updated.

**Comment by jaykayone** [comment]
OK, works fine

**Comment by jaykayone** [comment]
see also #631 because this PR does not entirely fix #591

File: geoportailv3/static/js/print/printdirective.js [hunk 1]
```diff
@@ -172,6 +175,11 @@ app.PrintController = function($scope, $timeout, $q, gettextCatalog,
   this['title'] = '';
 
   /**
+   * @type {boolean|undefined}
```
Comment (elemoine) [inline geoportailv3/static/js/print/printdirective.js]: `undefined`?

**Comment by elemoine** [comment]
LGTM. I just added a minor comment. Please merge when Travis build passes.

**Pull request merged**